lib/active_model/serializer.rb

```diff
@@ -188,7 +186,7 @@ def as_json(adapter_opts = nil)
 
     # Used by adapter as resource root.
     def json_key
-      root || _type || object.class.model_name.to_s.underscore
+      _type || object.class.model_name.to_s.underscore
```

Comment:
as https://github.com/rails-api/active_model_serializers/pull/1782/files#r66382756 I'm fine removing `root` as a public interface, or in the initializer, but it should still be an option to `as_json`